.pipelines/templates/template-push-images-to-acr.yml

```diff
@@ -12,3 +12,14 @@ steps:
     export BRANCH=$(Build.SourceBranchName)
     make publish-image-aro
   displayName: ⚙️ Build and push images to ACR
+- script: |
```

Comment:
At this point this is direct copy of the previous step. First the steps are duplicated, once all works old can be deleted.